activesupport/lib/active_support/core_ext/date/calculations.rb

```diff
@@ -1,12 +1,12 @@
-require 'date'
```

Comment:
It's already indirectly required by `require 'active_support/core_ext/date/zones'`